data/default.html

```diff
@@ -20,7 +20,55 @@
   </head>
   <body>
     <div id="container"></div>
-    <script src="https://www.youtube.com/iframe_api"></script>
+    <script>
+     // from youtube.com/iframe_api
```

Comment:
@meandavejustice Is there some reason you're inlining this, rather than create a vendor bundle? Just to save time, maybe?